### PR TITLE
Added alternative template option to seed create command

### PR DESCRIPTION
**Pull request by Firtzberg**
Adds the -t option to seed:create command, just like it exists for create command.

I've updated English documentation.

### CODE REVIEW

**Comment by Firtzberg** [comment]
Slightly related to https://github.com/cakephp/phinx/issues/1507

**Comment by Firtzberg** [comment]
Hi @dereuromark, is the failing 'codecov/project' OK?
I don't think I can solve it right now. I got the codecov/patch to 100%.

File: src/Phinx/Console/Command/SeedCreate.php [hunk 1]
```diff
@@ -166,8 +169,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
 
-        // inject the class names appropriate to this seeder
-        $contents = file_get_contents($this->getSeedTemplateFilename());
+        // Get the alternative template option from the command line.
+        $altTemplate = $input->getOption('template');
+
+        // Verify the alternative template file's existence.
+        if ($altTemplate && !is_file($altTemplate)) {
+            throw new InvalidArgumentException(sprintf(
+                'The alternative template file "%s" does not exist',
```
Comment (dereuromark) [inline src/Phinx/Console/Command/SeedCreate.php:178]:
```suggestion
                'The template file "%s" does not exist',
```

File: tests/Phinx/Console/Command/SeedCreateTest.php [hunk 2]
```diff
@@ -111,4 +111,63 @@ public function testExecuteWithInvalidClassName()
...
+    public function testAlternativeTemplateDoesntExist()
+    {
+        $application = new PhinxApplication();
+        $application->add(new SeedCreate());
+
+        /** @var SeedCreate $command */
+        $command = $application->find('seed:create');
+
+        /** @var Manager $managerStub mock the manager class */
+        $managerStub = $this->getMockBuilder('\Phinx\Migration\Manager')
```
Comment (dereuromark) [inline tests/Phinx/Console/Command/SeedCreateTest.php:156]: You can use ::class usage here.

Comment (Firtzberg) [reply tests/Phinx/Console/Command/SeedCreateTest.php]: done

File: src/Phinx/Console/Command/SeedCreate.php (hunk 1 above)
Comment (Firtzberg) [reply src/Phinx/Console/Command/SeedCreate.php]: done

**Comment by Firtzberg** [comment]
What is your release process like? Can you make a patch release? I need this feature

**Comment by dereuromark** [comment]
Yes, I think we can put this into the upcoming patch release.
@MasterOdin Does that sound OK to you?
Then let's do the patch tomorrow or so.

**Review by MasterOdin — APPROVED** [review]
I'm fine with a patch release.

**Pull request merged**